dev/preview/previewctl/pkg/k8s/context.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/imdario/mergo"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+)
+
+func RenameConfig(config *api.Config, oldName, newName string) (*api.Config, error) {
+	// TODO: add tests
```

Comment:
Nit: Lets just remove TODOs like this

Reply:
Replaced them with a link to https://github.com/gitpod-io/ops/issues/6524